components/reward-calculator/index.js

```diff
@@ -104,7 +104,7 @@ const RewardCalculatorPage = () => {
 		useStakingPathPopover();
 	const { setStakingPath } = useStakingPath();
 	const [loading, setLoading] = useState(false);
-	const [amount, setAmount] = useState(transactionState.stakingAmount || 1000);
+	const [amount, setAmount] = useState(transactionState.stakingAmount || 0);
```

Comment:
incase of no account selected it displays 0 amount as input instead of 1000
Amount inputs:
- if no account selected - 1000
- Account selected, no active bond -  max possible staking amount
- Account selected, active bond - Active bonded amount

Reply:
@sahilnanda1995 in case of account selected and account balance= 0. do you want max possible staking amount(i.e. 0) displayed or 1000?

We have also not covered the case of lowest possible stake of polkadot being 40 DOT.

Reply:
> We have also not covered the case of lowest possible stake of polkadot being 40 DOT.
yep, we need to take this into account too.

> in case of account selected and account balance= 0. do you want max possible staking amount(i.e. 0) displayed or 1000?
former(i.e. 0)